pkgs/by-name/js/jsql-injection/package.nix

```
@@ -0,0 +1,90 @@
+{
+  lib,
+  jre,
+  unzip,
+  stdenv,
+  fetchurl,
+
+  makeWrapper,
+  makeDesktopItem,
+  makeBinaryWrapper,
+
+  copyDesktopItems,
+  desktopToDarwinBundle,
+}:
+
+let
+  icon = fetchurl {
+    url = "https://www.kali.org/tools/jsql/images/jsql-logo.svg";
+    hash = "sha256-qjr+9gMGSdYqDT2SZFt4i3vISrMjqvAlBTt7U6raTLc=";
+  };
+
+  pname = "jsql-injection";
+  version = "0.111"; # ! Latest = v0.111
+in
+stdenv.mkDerivation {
+  inherit
+    pname
+    version
+    ;
+
+  src = fetchurl {
+    url = "https://github.com/ron190/jsql-injection/releases/download/v${version}/jsql-injection-v${version}.jar";
+    sha256 = "sha256-TQWhlD1xbW5Xxdy5yf6P6aAMHkodM4qEfP9CGRs73MA=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
```

prefer the binary wrapper when possible
```suggestion
    makeBinaryWrapper
```